src/modules/auth/auth.service.ts

```diff
@@ -195,13 +195,38 @@ export class AuthService {
     imageBuffer: Buffer,
     filename: string,
   ): Promise<PublicFile> {
```

Comment:
but you can get user from jwt validation middleware, why you need additional query?

Reply:
Made some changes so the JWT attaches all user information to the request object and removed the first find query. The first update query syncs the database with the S3 bucket and also removes the foreign key if there are any. The other uploads it to the cloud and updates the db.